File: 1-Towers-of-Hanoi/Towers-of-Hanoi.py
def hanoi(height, left='left', right='right', middle='middle'): # hight, from, to, using
    if height:
        hanoi(height - 1, left, middle, right)
        print(left, '=>', right)
        hanoi(height - 1, middle, right, left)

hanoi(3)

File: 1-Towers-of-Hanoi/Towers-of-Hanoi.js
function hanoi(height, left='left', right='right', middle='middle'){ // height, from, to, using
    if(height){
        hanoi(height - 1, left, middle, right)
        console.log(left,'=>', right)
        hanoi(height-1, middle, right, left)
    }
}

hanoi(3)